ansible/devspace.yml

```diff
@@ -8,3 +8,15 @@
     - role: docker
     - role: versioncontrol-utils
     - role: devspace
+    - role: samba-client
+
+  tasks:
+
+    - name: Mount samba share
+      become: yes
+      mount:
+        fstype: cifs
+        name: /ome/data_repo
+        opts: "credentials=/etc/fstab.datarepo.creds,ro"
```

Comment:
@manics - yes, but I what I mean is that I'm not sure if it provides any more options, like `domain`, `uid` or `gid` - i.e. the diff between what I see in the playbook, and what 'worked for me' on the cli.